tests/test_roms_marbl_example.py

```diff
@@ -37,9 +51,9 @@
     url_prefix="https://github.com/CWorthy-ocean/input_datasets_roms_marbl_example/raw/main/"
     if url_prefix in line:
         fileurl=line.split()[-1] # Just isolate URL from e.g. source: URL
-        filepath=f"{os.getcwd()}/local_input_files/{os.path.basename(fileurl)}"
-        if os.path.exists(filepath):
-            test_blueprint[i]=line.replace(fileurl,filepath)
+        filepath=Path(f"{Path.cwd()}/local_input_files/{Path(fileurl).name}")
```

Comment:
`pathlib.Path` understands the `/` operator, so you can literally just do this:
```suggestion
        filepath = Path.cwd() / 'local_input_files' / Path(fileurl).name
```

Reply:
just learned about this today, quite cool.